CMakeLists.txt

```diff
@@ -414,12 +414,28 @@ if(FEAT_TUI)
     return unibi_num_from_var(unibi_var_from_num(0));
   }
   " UNIBI_HAS_VAR_FROM)
-  unset(CMAKE_REQUIRED_INCLUDES)
-  unset(CMAKE_REQUIRED_LIBRARIES)
   if(UNIBI_HAS_VAR_FROM)
     add_definitions(-DNVIM_UNIBI_HAS_VAR_FROM)
   endif()
 
+  if(WIN32)
+    check_c_source_compiles("
+    #include <unibilium.h>
+
+    int
+    main(void)
+    {
+    unibi_set_fallback(NULL);
+    return 0;
+    }
+    " UNIBI_HAS_SET_FALLBACK)
+    if(UNIBI_HAS_SET_FALLBACK)
+      add_definitions(-DNVIM_UNIBI_HAS_SET_FALLBACK)
+    endif()
+  endif()
```

Comment:
It appears we have a minimum requirement of 2.0.0 already (or should fix the segfault mentioned in https://github.com/neovim/neovim/issues/10562#issuecomment-517922947).

Reply:
Ah, but that's even something even newer - I see.
Might be better to just bump the required version then though instead maybe?